lib/src/main/kotlin/net/dean/jraw/databind/RedditModelAdapterFactory.kt

```diff
@@ -166,6 +166,21 @@ class RedditModelAdapterFactory(
                         // "data"
                         listing = readListing(reader)
                     }
+                    2 -> {
```

Comment:
Yeah, thank you very much for pointing this out, on further investigation it is clear that this code doesn't belong there. I haven't had any experience with Moshi before so wasn't really sure how it works.

Creating a class with a single property `List<SimpleFlairInfo> users` doesn't look nice because we actually have a listing and allow pagination through it.

I have removed the code in question and instead provided serialization which is more in line with the library style in https://github.com/mattbdean/JRAW/pull/207/commits/40d9d77fa952530fe9724739bab3ebc29c9a0141